code/modules/projectiles/ammo_datums.dm

```diff
@@ -1657,7 +1657,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	bonus_projectiles_type = /datum/ammo/xeno/acid/heavy/scatter
 	bonus_projectiles_amount = 5
 	bonus_projectiles_scatter = 10
-	max_range = 7
```

Comment:
Undocumented change

Reply:
Yes, this is buckshot range; updated the description.

Not letting marines back peddle out of from the edge of their range, especially after we've seen that a defacto max range of 20 is not actually a problem.

Reply:
Add this to CL or remove the change